byond-extools/CMakeLists.txt

```diff
@@ -20,7 +20,11 @@ add_library(byond-extools SHARED ${SRC_FILES})
 source_group(TREE ${SRC_DIR} FILES ${SRC_FILES})
 if (WIN32)
     set_property(TARGET byond-extools PROPERTY MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
-    target_compile_options(byond-extools PRIVATE "/MP")
+    if (MSVC)
+        target_compile_options(byond-extools PRIVATE "/MP")
+    else()
+        target_link_libraries(byond-extools PRIVATE "ws2_32" "psapi")
```

Comment:
```suggestion
        target_link_libraries(byond-extools PRIVATE "ws2_32" "psapi" "-static-libgcc" "-static-libstdc++" "-static")
```

Reply:
I don't love this but it can be revisited later